medusa/providers/torrent/html/sdbits.py

```diff
@@ -0,0 +1,211 @@
+# coding=utf-8
+# Author: duramato
+#
+# This file is part of Medusa.
+#
+# Medusa is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Medusa is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Medusa. If not, see <http://www.gnu.org/licenses/>.
+"""Provider code for SDBits."""
+from __future__ import unicode_literals
+
+import datetime
+import re
+import traceback
+
+from pytimeparse import parse
+
+from requests.compat import urljoin
+from requests.utils import dict_from_cookiejar
+
+from ..torrent_provider import TorrentProvider
+from .... import logger, tv_cache
+from ....bs4_parser import BS4Parser
+from ....helper.common import convert_size, try_int
+from ....indexers.indexer_config import mappings
+
+
+class SDBitsProvider(TorrentProvider):
+    """SDBits Torrent provider."""
+
+    def __init__(self):
+        """Initialize the class."""
+        super(self.__class__, self).__init__('SDBits')
+
+        # Credentials
+        self.username = None
+        self.password = None
+
+        # URLs
+        self.url = 'http://sdbits.org'
+        self.urls = {
+            'login': urljoin(self.url, 'takeloginn3.php'),
+            'search': urljoin(self.url, 'browse.php'),
+        }
+
+        # Proper Strings
+        self.proper_strings = ['']
```

Comment:
There are no propers and such, and if there are a normal search returns them. No need for 3 more searches.